src/services/InMemoryGroupService.ts

```diff
@@ -0,0 +1,149 @@
+import { Group, IGroup } from '@lfg/types';
+import { User } from 'discord.js';
+import * as moment from 'moment';
+import { generate as generateShortId } from 'shortid';
+import IGroupService, { IGroupFilters } from './IGroupService';
+
+export default class InMemoryGroupService implements IGroupService {
+  /**
+   * In memory cache that keeps a mapping from a groups ID to the group definition
+   */
+  private readonly groupCache = new Map<string, Group>();
+
+  /**
+   * Get the group with the given ID
+   * @param groupId The ID of the group to get
+   */
+  public getGroup(groupId: string) {
+    const group = this.groupCache.get(groupId);
+
+    if (!group) {
+      throw new Error(`The group with ID ${groupId} does not exist`);
```

Comment:
Default functionality of the siegebot-client is to [log the error and fail silently to the user](https://github.com/siege918/siegebot-client/blob/master/index.js#L142-L144)

It would be up to the implementations in the commands folder to handle error messages sent to users. They currently don't make any effort to do that in most cases, but that can change.